<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LConsoleWidget</class>
 <widget class="QWidget" name="LConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="consoleEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
